src/main/java/com/microsoft/sqlserver/jdbc/IOBuffer.java

```diff
@@ -1826,6 +1826,16 @@ else if (con.getTrustManagerClass() != null) {
             // SSL is now enabled; switch over the channel socket
             channelSocket = sslSocket;
 
+            // Check the TLS version
+            String tlsProtocol = sslSocket.getSession().getProtocol();
+            if (SSLProtocol.TLS_V10.toString().equalsIgnoreCase(tlsProtocol)
+                    || SSLProtocol.TLS_V11.toString().equalsIgnoreCase(tlsProtocol)) {
```

Comment:
I think it might be better to put add a isMinTLS() (or something) method in SQLServerDriver? That way when a new version comes out it can be changed in 1 place only.

Reply:
this is the only place we check tls version.